mass_mailing_sending_queue/models/mail_mass_mailing_sending.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import api, fields, models, tools
+
+_logger = logging.getLogger(__name__)
+
+
+class MailMassMailingSending(models.Model):
+    _name = 'mail.mass_mailing.sending'
+
+    state = fields.Selection([
+        ('draft', "Draft"),
+        ('enqueued', "Enqueued"),
+        ('sending', "Sending"),
+        ('sent', "Sent"),
+        ('error', "Error"),
+    ], string="State", required=True, copy=False, default='enqueued')
+    mass_mailing_id = fields.Many2one(
+        string="Mass mailing", comodel_name='mail.mass_mailing', readonly=True)
+    pending = fields.Integer(
```

Comment:
This seems to be more a boolean than an integer. Use the suffix `_count` for noting the number